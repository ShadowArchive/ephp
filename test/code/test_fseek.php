<?php

$f = fopen("/tmp/test_stream_fseek", "w+");
var_dump(fwrite($f, "hello world!\n"));
var_dump(fseek($f, 0));
var_dump(fwrite($f, "Hallo"));
var_dump(fseek($f, 0));
$data = fread($f, 20);
var_dump($data);
fclose($f);

