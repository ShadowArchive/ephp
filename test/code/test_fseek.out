int(13)
int(0)
int(5)
int(0)
string(13) "Hallo world!
"
